### PR TITLE
change default arch value and remove couch patch

### PR DESCRIPTION
**Pull request by ticoann**
@amaltaro, please review

### CODE REVIEW

File: bin/deploy-wmagent.sh [hunk 1]
```diff
@@ -191,12 +191,6 @@ unzip -q deployment.zip
 cd deployment-$CMSWEB_TAG
 set +e 
 
-### Patching Couchdb1.6
-echo -e "\n*** Applying deployment patches (couchdb1.6, etc) ***"
-wget -nv https://github.com/dmwm/deployment/pull/162.patch -O - | patch -p 1
-chmod 600 /data/certs/service{cert,key}.pem
```
Comment (amaltaro) [inline bin/deploy-wmagent.sh]: I would keep this line to make sure certs are with the right permissions, otherwise couch1.6 will crash.

**Comment by amaltaro** [comment]
Sounds good! I kept the couch patch because once I start deploying agents with an HG deployment tag, I don't like changing it.

File: bin/deploy-wmagent.sh (hunk 1 above)
Comment (ticoann) [inline bin/deploy-wmagent.sh]: Hmm, I thought this was affecting only central site... or is it vice versa? When I ran the test without it wasn't make couch crash. Could you check as well?

Comment (amaltaro) [inline bin/deploy-wmagent.sh]: I had problems with servicekey.pem set to 400 in the past, for local agent.

**Comment by ticoann** [comment]
@amaltaro, I updated the patch leaving the file permission change. (I think you are right my permission was already changed for test before the script was run) Maybe that is why it worked. :smile: 
Please review.

**Comment by ticoann** [comment]
> Sounds good! I kept the couch patch because once I start deploying agents with an HG deployment tag, I don't like changing it.

Sorry I don't follow, current tag contains that couch patch, doesn't it?

**Comment by amaltaro** [comment]
Yes, current HG tag contains that. When I started deploying the 1.0.5 agents, I was using 1504x tag which did not have the patch in, since new changes got in as well, I didn't change the HG tag for deploying the same agent version.

Summary :-) feel free to remove the patch here. It would need to be removed anyways in a week or two :)

**Comment by amaltaro** [comment]
BTW, since you are updating it, can you update line 43 to (since we still don't have comp for slc6 scram)? 
REPO="comp=comp.pre"

Thanks!

**Comment by ticoann** [comment]
Thanks Alan, It is updated now. I will merge that patch then.

**Pull request merged**